lib/file-info-view.coffee

```diff
@@ -10,10 +12,30 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = ->
+      atom.clipboard.write(@getActiveItem()?.getPath?())
+      setTimeout =>
+        @newTooltip()
+      , 2000
+
+    @addEventListener('click', clickHandler)
+    @clickSubscription = new Disposable => @removeEventListener('click', clickHandler)
+
+  newTooltip: ->
```

Comment:
Not sure. I am used to such conventions from work, but I am not going to enforce that on anyone. Was just wondering as other methods have such names - `intialize`, `destroy`, `subscribe...`, `add...`, `remove...`, etc.